Bottom: 2c520ae51dc46fca7c9f93a65bdf8ac53aa17f4c
Top:    6bcd4aaee0bcadf083ace213f38b7ed01bf00b28
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 10:31:42 -0500

Enable file-based secrets provider


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index f9845106..8f045992 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -5,7 +5,7 @@
 # 'master.cfg' in your buildmaster's base directory.
 
 import os
-from buildbot.plugins import steps,util
+from buildbot.plugins import secrets,steps,util
 
 os.environ['LONG_RUN_TIMEOUT']='14400'
 
@@ -22,6 +22,9 @@ c['change_source'] = []
 # --master option)
 c['protocols'] = {"pb": {"port": int(os.getenv('BUILDBOT_WORKER_PORT',9989))}}
 
+### SECRETS MANAGEMENT
+c['secretsProviders'] = [secrets.SecretInAFile(dirname="/run/secrets")]
+
 ####### STATUS TARGETS
 
 # 'status' is a list of Status Targets. The results of each build will be
